### DISCUSSION TITLE
CSS

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#17] [Progress Tracker]
- Introduction to CSS.
- Use of CSS.
- Types of CSS
    - Inline
    - Document
    - External
- Selectors
    - Universal Selector
    - Class Selector
    - Id Selector
    - Element selector
    - Attribute Selector
    - Combinators
    - Pseudo selectors
- Properties:
    - Font
        - font-size
        - font-family
        - font-weight
        - font-style
    - Text
        - Text Decoration
            - text-decoration-line
            - text-decoration-style
            - text-decoration-color
        - text-align
        - text-transform
    - Border
        - Border Radius
            - border-top-left-radius
            - border-top-right-radius
            - border-bottom-right-radius
            - border-bottom-left-radius
        - Border Color
            - border-top-color
            - border-right-color
            - border-bottom-color
            - border-left-color
        - Border Style
            - border-top-style
            - border-right-style
            - border-bottom-style
            - border-left-style
    - Display
        - Flex
        - Grid
    - Position
        - absolute
        - fixed
        - sticky
        - relative
    - Background
        - background-color
        - image
    - height
    - width
    - transform

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by riya45810** [comment]
Completed.

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by ManikuttyBabu** [comment]
completed

**Comment by Shamla-P** [comment]
Completed

**Comment by AshinDamodaran** [comment]
completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by Nandana-04** [comment]
Completed

**Comment by thanoofnaina** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by kavitha-BS** [comment]
Completed

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by SivapriyaVB** [comment]
Completed
